src/Piral.Blazor.Tools/content/src/blazor.codegen

```diff
@@ -200,6 +206,9 @@ module.exports = async function () {
     return `    
     ${definePathCode}
     module.exports = {
+      setupPilet : (app) => {
+        ${setupFileExists ? `import("${setupFilePath}").then(e=> e.default(app))` : ''};
```

Comment:
Why lazy load it? This is immediately required. Use `require('${setupFilePath}').default`.

```js
setupPilet: ${setupFileExists ? `require('${setupFilePath}').default` : '() => {}'
```